config.yaml

```diff
@@ -2,6 +2,12 @@
 # See LICENSE file for licensing details.
 
 options:
+  synchronous_node_count:
+    description: |
+      Sets the number of synchronous nodes to be maintained in the cluster. Should be
+      either "all", "majority" or a positive integer value.
+    type: string
+    default: "all"
```

Comment:
For the history: it matches signed [spec](https://docs.google.com/document/d/1-a79fBYZPW2TUsKCGNyKjVWgLQefb9T7AiyV4QqW-y4/edit?tab=t.0).